### ISSUE TITLE
A11y_AzureDataStudio_New deployment - SQL Server on Windows_Non-Text Contrast: Color Contrast ratio of keyboard focus on the fields like Minimize, Close, Help to Improve SQL server is 1.8:1 which is less than 3:1

### ISSUE THREAD

**Issue by Gayathri1501** [microsoft/azuredatastudio#18597]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG1,4.11; #Win32; #AINotInScope; #ColorContrastAnalyser;

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment"
4. In the Deployment options, select "SQL Server on Windows" card.
5. From the dropdown for Version under Options, select from the available options.
6. Click the "Select" button
7. This will start the setup installer for SQL Server setup.
8. Follow the instructions through the setup process to install a SQL Server on Windows.
9. Verify whether the Non-text fields like Minimize, Close, Help to Improve SQL server have minimum contrast or not.

### Actual Result:
Color Contrast ratio of keyboard focus on fields like Minimize, Close, Help to Improve SQL sever is 1.8:1 which is less than 3:1

### Expected Result:
Color Contrast ratio of keyboard focus on fields like Minimize, Close, Help to Improve SQL server should be >=3:1
### Attachment:
 <img width="611" alt="On focus " src="https://user-images.githubusercontent.com/99962579/156167795-b7d430f4-ee6d-4f72-a3ac-ad84d96fe72b.png">

**Comment by alanrenmsft** [comment]
the install SQL server feature you tested is not a part of AzureDataStudio. so closing it.

**Comment by Gayathri1501** [comment]
@alanrenmsft --Could you please route it to the appropriate owner or team and reactivate the bug.
#Note: "Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"

**Comment by VijayaBharathyA** [comment]
#Closed;
Verified the issue in SQL 2022 environment :
![image](https://user-images.githubusercontent.com/100189407/219012904-238e2ece-c62d-469f-9df1-473e63759b3f.png)
